From: Matthias Clasen <mclasen@redhat.com>
Date: Thu, 1 Oct 2020 15:08:50 -0400
Subject: Require gtk-doc 1.33

We'll hopefully get a gtk-doc release out with that
version number soon.

Origin: upstream, 3.99.3, commit:ffb51b6888f9b199f31434cfeca89203130471c7
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 019b608..4476488 100644
--- a/meson.build
+++ b/meson.build
@@ -382,7 +382,7 @@ graphene_dep   = dependency('graphene-gobject-1.0', version: graphene_req,
                             fallback: ['graphene', 'graphene_dep'])
 iso_codes_dep  = dependency('iso-codes', required: false)
 
-gtk_doc_dep    = dependency('gtk-doc', version: '>=1.99',
+gtk_doc_dep    = dependency('gtk-doc', version: '>=1.33',
                             fallback: ['gtk-doc', 'dummy_dep'],
                             default_options: ['tests=false', 'yelp_manual=false'],
                             required: get_option('gtk_doc'))
